lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -307,13 +311,11 @@ def verify_file(self, path):
         Verify plugin configuration file and mark this plugin active
         Args:
             path: Path of configuration YAML file
-
         Returns: True if everything is correct, else False
-
         """
         valid = False
         if super(InventoryModule, self).verify_file(path):
-            if path.endswith(('vmware.yaml', 'vmware.yml')):
+            if path.endswith(('vmware.yaml', 'vmware.yml', 'vmware_vm_inventory.yaml', 'vmware_vm_inventory.yml')):
```

Comment:
When `auto` is used, `verify_file` is minimally helpful. But if someone has whitelisted this inventory plugin it will be more efficient than just trying to use it to parse all files which exist and are accessible (which is what the [base class `verify_file`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/__init__.py#L181) does).

Please update the documentation `- Uses any file which ends with vmware.yml or vmware.yaml as a YAML configuration file.` with these new file endings that are supported in 2.10+